CleanCode.Naming/RectangleClass.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace CleanCode.Naming
+{
+  class RectangleClass
+    {
+        private int firstSide;
+        private int secondSide;
+        private DateTime currentDate;
+
+        public RectangleClass(int firstSide, int secondSide)
+        {
+            this.firstSide = firstSide;
+            this.secondSide = secondSide;
+            currentDate = DateTime.Now;
+        }
+
+        public int getField()
+        {
+            return this.firstSide * this.secondSide;
+        }
+
+        // If duration should be in seconds
+        public int durationOfExistence(bool inMilliseconds)
```

Comment:
Może mozna byloby stworzyć 2 metody jedna zwracać bedzie milisekundy druga sekundy. Console.WriteLine(rectangle.durationOfExistence(false)); po takim wywolaniu patrzac tylko na to nie wiem jaki bedzie wynik musimy sprawdzic czym jest ten false.